Libraries/CO.CDP.AwsServices/AwsConfiguration.cs

```diff
@@ -5,10 +5,16 @@ public record AwsConfiguration
     public Credentials? Credentials { get; init; }
     public string? ServiceURL { get; init; }
     public Buckets? Buckets { get; init; }
-    public string? LogGroup { get; init; }
-    public string? LogStream { get; init; }
+
     public SqsDispatcherConfiguration? SqsDispatcher { get; init; }
     public SqsPublisherConfiguration? SqsPublisher { get; init; }
+    public CloudWatch? CloudWatch { get; init; }
+}
+
+public record CloudWatch
+{
+    public string? LogGroup { get; init; }
+    public string? LogStream { get; init; }
```

Comment:
Since we grouped these two under CloudWatch, we can make these non-nullable (CloudWatch above in AwsCredentials remains nullable).